gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -191,6 +192,12 @@ public final void createConfig() throws IOException {
                         .addAllJvmOpts(javaAgentArgs())
                         .addAllJvmOpts(alwaysOnJvmOptions)
                         .addAllJvmOpts(addJava8GcLogging.get() ? java8gcLoggingOptions : ImmutableList.of())
+                        // Java 11u6 and later have potential memory leak issues when using the C2
```

Comment:
I'm not familiar with u-based versioning in newer java releases (google search doesn't give me many hits for it)
```suggestion
                        // Java 11.0.16 introduced a potential memory leak issues when using the C2
```